CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ FEATURES:
 * `vcd_vapp_vm` - Ability to add metadata to a VM. For previous behaviour please see `BACKWARDS INCOMPATIBILITIES` [GH-158]
 * `vcd_vapp_vm` - Ability to enable hardware assisted CPU virtualization for VM. It allows hypervisor nesting. [GH-219]
 * **New Resource:** external network - `vcd_external_network` - [GH-230]
+* **New Resource:** VDC resource `vcd_vdc` - [GH-234]
```

Comment:
I believe we should name this resource "Organization VDC resource `vcd_org_vdc`". Reasoning:
- Reflect clearly that this is **Org** VDC and _not_ **PVDC**.
- Keep the name for PVDC implementation in the future. Even though we may call it `vcd_provider_vdc`, we need to clearly separate both.
- We have "Organization VDCs" in the GUI.
- `vcd_vdc` sounds to me highest VDC level, which intuitively implies that this is PVDC, which is not.
- We already have `vcd_org` resource, so having `vcd_org_vdc` sounds as a natural extension.

Reply:
I vote for "vcd_org_vdc"

Reply:
Changed